src/pip/_internal/commands/install.py

```diff
@@ -737,4 +738,17 @@ def create_os_error_message(error, show_traceback, using_user_site):
             parts.append(permissions_part)
         parts.append(".\n")
 
+    # Suggest the user to enable Long Paths if path length is
+    # more than 260
+    if (WINDOWS and error.errno == errno.ENOENT and error.filename and
+            len(error.filename) > 260):
+        parts.append(
+            "A potential cause to this error is "
+            "Long Paths being disabled on your system. "
+            "Please set LongPathsEnabled to 1 in the "
+            "registry and try again. For further instructions "
+            "please refer to the documentation: "
+            "https://pip.pypa.io/warnings/enable-long-paths"
```

Comment:
Suggested rephrasing, to avoid mentioning the registry. This is a non-blocker for the PR though, since we can do this later as well:

```suggestion
            "HINT: This error might have occurred since "
            "this system does not have Windows Long Path "
            "support enabled. You can find information on "
            "how to enable this at "
            "https://pip.pypa.io/warnings/enable-long-paths. "
```

Reply:
The newline at the end makes it easier for us to add more parts to this message at a later date.

Reply:
I will get it rephrased.